File: Palindrome.rb
inputOne = 'racecar'
inputTwo = 'nickname'

def get_reversed_word(word)
    reversedWord= ''
    
    for i in 0...word.length do
        reversedWord = word[i] + reversedWord
    end
    
    return reversedWord
    
end

def is_palindrome(input)
    reversedWord = get_reversed_word(input)
    return input == reversedWord
    
end

puts is_palindrome(inputOne)
puts is_palindrome(inputTwo)

File: Palindrome.js
 
 var inputOne = 'racecar';
 var inputTwo = 'nickname';
 
 function reverse (word){
     
     var newReverseWord = '';
     
     for (var i = word.length - 1; i > -1; i = i-1){
         newReverseWord = newReverseWord + word[i];
     }
     console.log(newReverseWord);
     return newReverseWord;
 }
 function isPalindrome (input){
     var reverseWord = reverse(input);
     
     if (inputOne === reverseWord) {
         return true;
     } else {
         return false;
     }
 }
 
 console.log(isPalindrome(inputOne));
 console.log(isPalindrome(inputTwo));
